rcl/test/rcl/test_time.cpp

```diff
@@ -533,21 +499,15 @@ TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), rcl_time_fail_set_jump_callbacks)
 }
 
 TEST(CLASSNAME(rcl_time, RMW_IMPLEMENTATION), rcl_time_forward_jump_callbacks) {
+  rcl_clock_t ros_clock;
   rcl_allocator_t allocator = rcl_get_default_allocator();
-  auto * ros_clock =
-    static_cast<rcl_clock_t *>(allocator.allocate(sizeof(rcl_clock_t), allocator.state));
-  OSRF_TESTING_TOOLS_CPP_SCOPE_EXIT(
-  {
-    allocator.deallocate(ros_clock, allocator.state);
-  });
-  rcl_ret_t retval = rcl_ros_clock_init(ros_clock, &allocator);
-  ASSERT_EQ(retval, RCL_RET_OK) << rcl_get_error_string().str;
+  rcl_ret_t ret = rcl_ros_clock_init(&ros_clock, &allocator);
+  ASSERT_EQ(ret, RCL_RET_OK) << rcl_get_error_string().str;
```

Comment:
ros_clock doesn't need to be cleaned up if ret is not OK?

Reply:
It doesn't. And I _think_ we do strive for that to be the case for every `rcl` "object" (and rightly so).